### PR TITLE
Fixes the okta widget visual bug(content overflow) in styles.less

### PR DESCRIPTION
**Pull request by eet349**
The purpose of the PR is to fix a visual bug that would hide some content of the okta widget if a user entered in the wrong credentials. [Trello card ](https://trello.com/c/LndeSTyk)

### CODE REVIEW

**Comment by eet349** [comment]
The diffs show that there were more changes than there actually were. Prettier made those changes.

File: src/styles/styles.less [hunk 1]
```diff
@@ -10,7 +10,6 @@
 }
 
 #okta-sign-in.auth-container.main-container {
-  height: 530px;
   overflow: hidden;
```
Comment (L-Steinmacher) [inline src/styles/styles.less:13]: Ahh i see you deleted the fixed pixel height!

**Review by L-Steinmacher — APPROVED** [review]
Looks like prettier changed a number of lines.  The file looks almost identical to the original.

**Review by JackHenry15 — APPROVED** [review]
This looks good. We needed to fix the visual bug- approving this to push it up

**Pull request merged**